final_weather_concatentation_script.ipynb

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.                    if not os.path.exists(f'./data/final-weather/{curr_year}/week_{week}'):
</pre><p>change the file path to <code>f'./data/final-weather/{curr_year}/week_{week}'</code></p><p> </p><p>apply for the bottom two as well </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>